README.md

```diff
@@ -1,4 +1,5 @@
 # ride
+[![Build Status](https://semaphoreci.com/api/v1/fmmrtns/ride/branches/master/badge.svg)](https://semaphoreci.com/fmmrtns/ride)
```

Comment:
This points to `fmmrtns/ride` not `LoveNationArmy/ride`. Is this intentional or should we change it?